README.md

```diff
@@ -77,3 +77,10 @@ Be sure to `npm run build` and commit before each push to master. A few seconds
 * [Mocha](https://mochajs.org/)
 * [Chai](https://chaijs.com/)
 
+
+## User login? Cookies? JSON web tokens
+## Any idea why heroku pg isn't working as expected?
+## What else other than buttons opening up more stuff?
+## Code organization?
+## 3 endpoints?
+## ok to always work off real API (heroku)?
```

Comment:
Not for a real project, you’d want a few different environments. Probably do you development locally and hit local host, then promote it to a staging environment to make sure it doesn’t break things other people were working on, then it could be production.